README.md

```diff
@@ -33,24 +33,23 @@ If you'd like, make sure to include the following comment code tag in your READM
 build and deploy your sample:
 
 ```text
-[//]: # ({sst-run-unix})
+[//]: # ({sst-run-bash})
 ```
 
 For example:
 ````text
-[//]: # ({sst-run-unix})
+[//]: # ({sst-run-bash})
 ```
 gcloud builds submit --tag=gcr.io/${GOOGLE_CLOUD_PROJECT}/run-mysql
 ```
 ````
 
 When parsing the README for custom build and deploy commands, the serverless sample tester will include any commands
-inside a code fence that is immediately preceded by a line containing `{sst-run-unix}`. You can use Markdown syntax
-(e.g. `[//]: # ({sst-run-unix})`) to include this line without making it visible when rendered.
+inside a code fence that is immediately preceded by a line containing `{sst-run-bash}`. You can use Markdown syntax
+(e.g. `[//]: # ({sst-run-bash})`) to include this line without making it visible when rendered.
 
-The parsed commands will not be run through a shell, meaning that the program will not perform any expansions,
-pipelines, redirections or any other functions that shells are responsible for. This also means that popular shell
-builtin commands like `cd`, `export`, and `echo` will not be available or may not work as expected.  
+The parsed commands will be run through the Bash shell, but each command will be run through a separate instance
+of the shell, meaning that commands such as `cd` or `export` may not work as expected.  
```

Comment:
Can you clarify how users are supposed to do control dir of execution or env variables, then? Can I chain commands via `&&` for example?

Reply:
Thank you for your comment.

For environment variables, each command's process will uses the environment of the process from which users execute our program. So environment variables they have set in their shell where they call `sst` will be available for the commands that they write in their README. I'm going to adding a comment in our README to clarify this.

For the directory of execution, we're not currently supporting the option for users to control the directory where their README commands are executed; all commands will be executed in the sample's root directory (I going to a comment in our README to clarify this as well). But we're going to open an issue for this feature to make sure that it's tracked.

However, as you're alluding to, you are able to get around this by chaining commands together, like `cd .. && ls`. But 
I think we're not going to put this in our documentation because we believe a more permanent solution is preferred, and since each command is run in it's own separate instance of a shell, so you would have to chain each where you want a directory command with a `cd`.

Hope this makes sense. Did this answer your questions?